packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -696,10 +700,14 @@ def check_executor_metrics_metadata():
         check_executor_metrics_metadata()
 
 
-@pytest.mark.supportedwindows
 def test_metrics_node(dcos_api_session):
     """Test that the '/system/v1/metrics/v0/node' endpoint returns the expected
     metrics and metric metadata.
+
+    This test is not supported on Windows since dcos_metrics
+    plugin is not supported on Windows
+    https://github.com/dcos/telegraf/blame/d3666e/plugins/outputs/dcos_metrics/README.md#L5
```

Comment:
we mave metrics on windows, see here https://docs.google.com/document/d/1D7K6X8CDIiLkl66f6NqFMDZLGgoBU4QQsB2eT8Psonk/edit#heading=h.ev9f07y3dpze